site/content/install/cli/binary.md

```diff
@@ -0,0 +1,27 @@
++++
+title = "Install GetEnvoy Binary"
+environment = "Binary"
+type = "install"
+tags = []
+categories = []
+logo = "/images/linux.png"
++++
+
+## Requirements ##
+
+GetEnvoy is supported on macOS and Linux.
+
+## Installation ##
+
+1. **Install CLI.**
+```sh
+$ curl -L https://getenvoy.io/install | bash -s -- -b /usr/local/bin
+```
+{{% notice warning %}}
+If you install to a different location, ensure the binary is in your `$PATH` environment variable.
+{{% /notice %}}
+
+1. **Verify GetEnvoy is installed.**
+```sh
+$ getenvoy --help
```

Comment:
nit: Should we use `getenvoy version` here or similar rather than `help`? (do we have a `version` command?)